### ISSUE TITLE
Release to Kinetic

### ISSUE THREAD

**Issue by davetcoleman** [RethinkRobotics/baxter_common#80]
Even though I understand a full official release of Baxter for Kinetic has its own timeline, delays, and internal Rethink testing requirements, I think you guys should go ahead and released baxter_description into Kinetic so that, at least in simulation, Baxter can be easily used in the latest LTS. We also need this in the moveit_robots repo for Kinetic, where baxter_moveit_config lives. Currently our Travis build for that repo is failing because of that missing dependency.

**Comment by IanTheEngineer** [comment]
baxter_simulator and baxter_moveit_config need to be tested (and updated) to release in Kinetic. The current development focus is Sawyer SDK. Can we decouple baxter_moveit_config from moveit_robots to unblock releasing the other robots into Kinetic?

**Comment by davetcoleman** [comment]
Yes, but releasing just the URDF package should be no problem - like 10 minutes of using bloom.

**Comment by knorth55** [comment]
Most of baxter users use indigo, and RethinkRobotics officially supports indigo, hydro and groovy.
Is it necessary to release Kinetic `baxter_moveit_config` package?
There are few baxter users with kinetic, and it is not officially recommended.
The idea to separate `baxter_moveit_config` from `moveit_robots` sounds good to me (it's just an opinion)

**Comment by davetcoleman** [comment]
I'm only suggesting to release baxter_description.

I've been using Baxter on Kinetic for 4 months now, with MoveIt! - it works great https://www.youtube.com/watch?v=Tw3qOeOAKlc

**Comment by knorth55** [comment]
i didn't know that some baxter use kinetic version.
it was my misunderstanding, i'm sorry for that.
(BTW, your video is really cool!)

**Comment by haz** [comment]
Any tips on getting baxter to work with moveit on kinetic? I've managed to get the latest sdk compiled and can use the standard rethink tools to control the robot, but the simple demos (e.g., `roslaunch baxter_moveit_config demo_baxter.launch`) fail because of pervasive collisions. A same initial / goal state plan request gives the following in the rviz status:

```
Changed goal state
Goal state colliding links:
collision_head_link_1 - display
collision_head_link_2 - display
l_gripper_l_finger - l_gripper_l_finger_tip
l_gripper_l_finger - left_gripper_base
l_gripper_r_finger - l_gripper_r_finger_tip
l_gripper_r_finger - left_gripper_base
r_gripper_l_finger - r_gripper_l_finger_tip
r_gripper_l_finger - right_gripper_base
r_gripper_r_finger - r_gripper_r_finger_tip
r_gripper_r_finger - right_gripper_base
```

**Comment by IanTheEngineer** [comment]
Yep, that's probably not a Kinetic thing, but rather how your launching it. I'd recommend following the tutorial & adding grippers:
http://sdk.rethinkrobotics.com/wiki/MoveIt_Tutorial#Tutorial
```
     $ roslaunch baxter_moveit_config baxter_grippers.launch
```
or
```
     $ roslaunch baxter_moveit_config demo_baxter.launch right_electric_gripper:=true left_electric_gripper:=true
```

**Comment by haz** [comment]
I thought for sure that I tried the second, but apparently not. Apologies
for the noise, and thanks for the tip!
